bin/logstashScripts/logstash.conf

```diff
@@ -1,26 +1,18 @@
 input {
     file {
-        path => "{ELK_SRC_HOME}**/*.java"
+	path => "{ELK_SRC_HOME}/**/*.java"
```

Comment:
use <ELK_SRC_HOME> instead.

Reply:
the indentation also needs to be fixed

Reply:
placeholder has been changed and indentation fixed